mypy.ini

```diff
@@ -2,3 +2,6 @@
 python_version = 3.6
 warn_unused_configs = True
 exclude = venv/bin/
+
+[mypy-psutil.*]
+ignore_missing_imports = True
```

Comment:
It works for me, it seems like a local problem on your side. Maybe `types-psutil` isn't installed correctly? It appears in `dev-requirements.txt`.
Try to remove this and run the CI again. If it passes I don't think we should add these lines.